src/scripts/components/foundations/SVGIcon.js

```diff
@@ -11,20 +11,22 @@ type Props = {
 
 export const IconFillStyleColor = css`
   ${props => {
-    let _color: String
+    let _fill: String
 
     if (props.color === 'white') {
-      _color = props.theme.colors.button.white
+      _fill = props.theme.colors.button.white
     } else if (props.color === 'purple') {
-      _color = props.theme.colors.button.purple
+      _fill = props.theme.colors.button.purple
     } else if (props.color === 'pink') {
-      _color = props.theme.colors.button.pink
+      _fill = props.theme.colors.button.pink
+    } else if (props.color === 'green') {
+      _fill = props.theme.colors.button.green
     } else {
-      _color = props.theme.colors.button.gray
+      _fill = props.theme.colors.button.gray
     }
```

Comment:
I think it would be much easier to not require passing in of a `string` color. Can't we just pass in the color value directly? Why restrict like this?

Reply:
Could we change that in another PR?

Reply:
Sounds good to me 👍